fuzz/run.sh

```diff
@@ -2,31 +2,45 @@
 
 handle_fail()
 {
-  let "fails++"; mkdir "fail-$fails" ; cat out ; echo "Failed instance number $fails" >> fails ; cat out >> fails ; mv out "fail-$fails" ;
+  let "fails++"; mkdir "fail-${fails}" ; sed -i "1iFailed instance number ${fails}, with password '$password'" out ; cat out ; cat out >> fails ; mv out "fail-${fails}" ;
 }>&2
 
 fails=0
 runs=0
-echo "" > fails
+echo -n "" > fails
 while true
  do
+   password=""
    radamsa test_mnemonic_str > mnemonic_str
    radamsa test_entropy_bin > entropy_bin
    radamsa test_entropy_hex > entropy_hex
    radamsa test_seed_bin > seed_bin
    radamsa test_seed_hex > seed_hex
 
    mnemoniccli -g -e entropy_hex -s seed -m mnemonic -ll debug &>out
-   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp entropy_hex "fail-$fails" ; fi
+   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp entropy_hex "fail-${fails}" ; fi
    mnemoniccli -g -e entropy_bin -s seed -m mnemonic -f bin -ll debug &>out
-   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp entropy_bin "fail-$fails" ; fi
+   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp entropy_bin "fail-${fails}" ; fi
    mnemoniccli -r -m mnemonic_str -s seed -e entropy -ll debug &>out
-   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp mnemonic_str "fail-$fails" ; fi
+   if [[ $? != 0 && $? != 65 ]] ; then handle_fail ; cp mnemonic_str "fail-${fails}" ; fi
    mnemoniccli -v -m mnemonic_str -s seed_hex -ll debug &>out
-   if [[ $? != 0 && $? != 65 && $? != 125 ]] ; then handle_fail ; cp mnemonic_str "fail-$fails" ; cp seed_hex "fail-$fails" ; fi
+   if [[ $? != 0 && $? != 65 && $? != 125 ]] ; then handle_fail ; cp mnemonic_str "fail-${fails}" ; cp seed_hex "fail-${fails}" ; fi
    mnemoniccli -v -m mnemonic_str -s seed_bin -f bin -ll debug &>out
-   if [[ $? != 0 && $? != 65 && $? != 125 ]] ; then handle_fail ; cp mnemonic_str "fail-$fails" ; cp seed_bin "fail-$fails" ; fi
-   runs=$((runs + 5))
-   echo "Fails: $fails / $runs"
+   if [[ $? != 0 && $? != 65 && $? != 125 ]] ; then handle_fail ; cp mnemonic_str "fail-${fails}" ; cp seed_bin "fail-${fails}" ; fi
+
+   password=$(echo "password123" | radamsa)
```

Comment:
Will radamsa generate invalid utf-8 sequences?

Reply:
not sure, but i think its output is in ascii, so possibly yes. Do we care about that?

Reply:
Yes, because we need to test invalid UTF-8 in password, which is read as UTF-8 now. I guess our program will crash now, if you give it password as `\xe3\x00\x82`.

Reply:
didnt find an answer so I configured it to fuzz only password and got these fails - https://pastebin.com/ksVYG6B2, so I think it is ok

Reply:
Yes, looks like fuzzing is good, cause our program is bad with handling utf-8 password. :D  Thanks